maintainers/scripts/update.py

```diff
@@ -12,6 +12,7 @@
 
 class CalledProcessError(Exception):
     process: asyncio.subprocess.Process
+    stderr: bytes
```

Comment:
This should be Optional as in https://github.com/NixOS/nixpkgs/pull/194908

Reply:
Got it.